hi-ml/testhiml/testhiml/test_box_utils.py

```diff
@@ -8,27 +8,72 @@
 from health_ml.utils.box_utils import Box, get_bounding_box
 
 
-def test_get_bounding_box() -> None:
+def test_no_zeros() -> None:
     length_x = 3
     length_y = 4
     # If no elements are zero, the bounding box will have the same shape as the original
-    mask = np.random.randint(1, 10, size=(length_x, length_y))
+    mask = np.random.randint(1, 10, size=(length_y, length_x))
     bbox = get_bounding_box(mask)
     assert isinstance(bbox, Box)
     assert bbox.w == length_x
     assert bbox.h == length_y
 
+
+def test_bounding_box_3d() -> None:
     # passing a 3D array should cause an error to be raised
-    length_z = 5
-    mask_3d = np.random.randint(0, 10, size=(length_x, length_y, length_z))
+    mask_3d = np.random.randint(0, 10, size=(1, 2, 3))
     with pytest.raises(TypeError):
         get_bounding_box(mask_3d)
 
+
+def test_identity_matrix() -> None:
     # passing an identity matrix will return a bounding box with the same shape as the original,
     # and xmin and ymin will both be zero
-    mask_eye = np.eye(length_x)
+    length = 5
+    mask_eye = np.eye(length)
     bbox_eye = get_bounding_box(mask_eye)
     assert isinstance(bbox_eye, Box)
-    assert bbox_eye.w == length_x
-    assert bbox_eye.h == length_x
+    assert bbox_eye.w == length
```

Comment:
[optional] As a `dataclass`, `Box` implements `__eq__()`. Might be a bit clearer and less verbose to instantiate `expected_box = Box(...)` and then `assert whatever_box == expected_box` (also in the other tests below).